docs/_templates/components_requirements.tmpl

```diff
@@ -1,39 +1,116 @@
+{% if data['module_compatibility_ranges']%}
 
-This table is only meant as a reference for developers of Inmanta extensions and is meant to be used as input to a
-dependency resolver e.g. pip to ensure the resulting versions are compatible.
-This information is not relevant when installing the orchestrator via RPM as is recommended
-in the :ref:`install-server` section.
 
-.. list-table:: Inmanta core and extensions versions
-   :header-rows: 1
+Module compatibility ranges
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   * - .. centered:: Component
-     - .. centered:: Required version
+.. only:: iso
 
-   {% for key, value in data["component_requirements"] | dictsort %}
-   * - .. centered:: {{ key }}
-     - .. centered:: {{ value }}
-   {% endfor %}
+    The continuous integration can be configured to run tests against multiple ISO versions. Each version might require
+    a different set of Inmanta modules and different versions of these modules.
 
-.. note::
+.. only:: oss
+
+    The continuous integration can be configured to run tests against multiple OSS versions. Each version might require
+    a different set of Inmanta modules and different versions of these modules.
+
+
+This configuration can be done by either:
+
+- Using a specific pip index (or git repos) per ISO version and making sure it only contains the compatible versions of the modules.
+- Using a generic pip index (or git repos) containing module versions for different ISO versions and adding a
+  constraint file for each ISO version in which the compatible module ranges are specified.
+
+
+{% set list_only = ["lsm", "std"] %}
+
+.. only:: iso
+
+    The table below shows the set of Inmanta modules that integrate tightly with version |version_major| of the
+    Inmanta Service Orchestrator and which versions are compatible with this ISO release.
 
-    This information is also available as a constraints file in `requirements.components.txt <./requirements.components.txt>`_
-    and in json format in the `compatibility.json <./compatibility.json>`_ file.
+    .. list-table:: Modules
+       :header-rows: 1
+
+       * - Module
+         - Version constraint
+
+       {% for key, value in data["module_compatibility_ranges"]  | dictsort %}
+       {% if key in list_only %}
+       * - {{ key }}
+         - .. centered:: {{ value }}
+       {% endif %}
+       {% endfor %}
+
+.. only:: oss
+
+    The table below shows the set of Inmanta modules that integrate tightly with this Inmanta OSS version |release| and which versions are compatible with this OSS release.
+
+    .. list-table:: Modules
+       :header-rows: 1
+
+       * - .. centered:: Module
+
+       {% for key, value in data["module_compatibility_ranges"] | dictsort %}
+       {% if key in list_only %}
+       * - {{ key }}
+       {% endif %}
+       {% endfor %}
+
+.. note::
 
+    This information is also available in a machine-consumable format in the `compatibility.json <./compatibility.json>`_ file.
```

Comment:
I would also mention the other modules, i.e. those that are not "most commonly used" as put above.